### ISSUE TITLE
RuntimeError - Cannot write 3.0 to an ascii plist

### ISSUE THREAD

**Issue by fanyuexiang** [CocoaPods/CocoaPods#6114]
> ℹ Please fill out this template when filing an issue.
> All lines beginning with an ℹ symbol instruct you with
> what info we expect.  
> Please remove this line and all above before submitting.

> Before you start, are you using the CocoaPods beta release?
> a lot has changed with Xcode 8 that is not backwards compatible.
>
> Xcode 8 Status: Only works with CocoaPods 1.1.0
> Swift 3: You need to add a `.swift-version` file, see the CHANGELOG
> China: Use https://gems.ruby-china.org/ instead of https://gems.ruby-china.org/ for the RC see #5910

* [ ] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

ℹ Please replace this with what you did.  
Run `pod install`

## What did you expect to happen?

ℹ Please replace this with what you expected to happen.  
 Install all pod dependencies correctly.

## What happened instead?

ℹ Please replace this with of what happened instead.  
a error is come out:
```
RuntimeError - Cannot write 3.0 to an ascii plist
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:63:in `write_object'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:140:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:41:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:119:in `block in write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `each'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:19:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:55:in `write_object'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:140:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:41:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:119:in `block in write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `each'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:19:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:55:in `write_object'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:140:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:41:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:29:in `block (2 levels) in write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:28:in `each'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:28:in `block in write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:24:in `each'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:24:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:55:in `write_object'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:140:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:41:in `write_dictionary_key_value_pair'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:119:in `block in write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `each'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:118:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/xcode_project_writer.rb:19:in `write_dictionary'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:55:in `write_object'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/nanaimo-0.1.2/lib/nanaimo/writer.rb:29:in `write'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/xcodeproj-1.4.0/lib/xcodeproj/project.rb:353:in `block in save'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/xcodeproj-1.4.0/lib/xcodeproj/project.rb:353:in `open'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/xcodeproj-1.4.0/lib/xcodeproj/project.rb:353:in `save'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/installer/xcode/pods_project_generator.rb:77:in `block in write'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/user_interface.rb:142:in `message'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/installer/xcode/pods_project_generator.rb:69:in `write'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/installer.rb:182:in `block in generate_pods_project'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/user_interface.rb:64:in `section'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/installer.rb:178:in `generate_pods_project'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/installer.rb:115:in `install!'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/command/install.rb:37:in `run'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/claide-1.0.1/lib/claide/command.rb:334:in `run'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/lib/cocoapods/command.rb:50:in `run'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/gems/cocoapods-1.1.0/bin/pod:55:in `<top (required)>'
/Users/fanyuexiang/.rvm/rubies/ruby-2.3.0/bin/pod:22:in `load'
/Users/fanyuexiang/.rvm/rubies/ruby-2.3.0/bin/pod:22:in `<main>'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/bin/ruby_executable_hooks:15:in `eval'
/Users/fanyuexiang/.rvm/gems/ruby-2.3.0@global/bin/ruby_executable_hooks:15:in `<main>'
```

## CocoaPods Environment

ℹ Please replace this with the output of `pod env`.

### Stack

```
   CocoaPods : 1.1.0
        Ruby : ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin16]
    RubyGems : 2.6.7
        Host : Mac OS X 10.12.1 (16B2657)
       Xcode : 8.1 (8B62)
         Git : git version 2.9.3 (Apple Git-75)
Ruby lib dir : /Users/fanyuexiang/.rvm/rubies/ruby-2.3.0/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 6da71ef4f538aabbb9964dbe147e0bbb2380988b
```

### Installation Source

```
Executable Path: /Users/fanyuexiang/.rvm/rubies/ruby-2.3.0/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.1
cocoapods-try         : 1.1.0
```

**Comment by DanToml** [comment]
@fanyuexiang Please could you share steps to reproduce this?

**Comment by segiddins** [comment]
This should be fixed with Xcodeproj 1.4.1
